api/enterprise-api-ref/host.markdown

```diff
@@ -295,6 +295,61 @@ HTTP 200 Ok
 }
 ```
 
+## Restore deleted hosts entry
+
+**URI:** https://hub.cfengine.com/api/hosts/restore-deleted/:host-id
+
+**Method:** POST
+
+Restore host entry in reporting database .
+
+Note: to abe able to perform this action related RBAC rule should be enabled.
```

Comment:
```suggestion
Note: to be able to perform this action related RBAC rule should be enabled.
```

Reply:
Do we want to add a reference link or actual name of RBAC rule either UI version and/or database format?

Reply:
I would guess `deletePermanently` and `restoreDeletedHosts`
https://github.com/cfengine/mission-portal/pull/2463/files#diff-00c4fcc2f1a37491edbfb456049471eebed9694282dd2f551010ab0b9877ab27R80